src/main/kotlin/com/cognifide/gradle/aem/api/AemConfig.kt

```diff
@@ -421,63 +423,82 @@ class AemConfig(
         instance(LocalInstance.create(httpUrl))
     }
 
-    fun localInstance(httpUrl: String, type: String) {
-        instance(LocalInstance.create(httpUrl, type))
+    fun localInstance(httpUrl: String, configurer: LocalInstance.() -> Unit) {
+        instance(LocalInstance.create(httpUrl, configurer))
     }
 
-    fun localInstance(httpUrl: String, user: String, password: String) {
-        instance(LocalInstance.create(httpUrl, user, password))
+    fun localInstance(httpUrl: String, configurer: Closure<*>) {
+        localInstance(httpUrl, { ConfigureUtil.configure(configurer, this) })
```

Comment:
@jplucinski note that for Groovy DSL this is still needed (for Kotlin DSL is not)